CONTRIBUTING.md

```diff
@@ -0,0 +1,194 @@
+# Contributing
+
+Thanks for taking the time to join our community and start contributing.
+These guidelines will help you get started with the Contour project.
+Please note that we require [DCO sign off](#dco-sign-off).  
+
+## Building from source
+
+This section describes how to build Contour from source.
+
+### Prerequisites
+
+1. *Install Go*
+
+    ir2proxy requires [Go 1.13][1] or later.
+    We also assume that you're familiar with Go's [`GOPATH` workspace][3] convention, and have the appropriate environment variables set.
```

Comment:
You don't need GOPATH because modules, right?

Reply:
Which means that most of the verbiage below is also unnecessary?

Reply:
Yeah, probably. But I lifted it from Contour's one, which hasn't been updated in ages. I'll remove it.

Reply:
I changed things around instead.